crates/hir-def/src/attr.rs

```diff
@@ -824,7 +827,27 @@ impl Attr {
             Some(Interned::new(AttrInput::Literal(value)))
         } else if let Some(tt) = ast.token_tree() {
             let (tree, map) = syntax_node_to_token_tree(tt.syntax());
-            Some(Interned::new(AttrInput::TokenTree(tree, map)))
+            if path.to_string() == "cfg_attr" {
```

Comment:
My brain cache around r-a is pretty stale, however I think you can avoid allocation here, since this path is just a single identifier right? Something like this: `path.as_ident().unwrap().as_text().unwrap()` maybe?